linkcheck/htmlutil/loginformsearch.py

```diff
@@ -44,10 +45,8 @@ def search_form(content, cgiuser, cgipassword):
     cginames = {cgiuser, cgipassword} - {None}
     for form_element in soup.find_all("form", action=True):
         form = Form(form_element["action"])
-        for input_element in form_element.find_all("input",
-                                                   attrs={"name": True}):
-            form.add_value(
-                input_element["name"], input_element.attrs.get("value"))
+        for input_element in form_element.find_all("input", attrs={"name": True}):
+            form.add_value(input_element["name"], input_element.attrs.get("value"))
```

Comment:
I like it when a single file's diff fits entirely on screen.

and my comments are getting increasingly offtopic, which is a sign that maybe I should stop reviewing for now.  36 out of 79 files.